examples/MG_redshift_only_dependence_examples.ipynb

```diff
@@ -0,0 +1,558 @@
+{
```

Comment:
This file should disappear from the repo before we merge.

Reply:
removed